Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: release
+release: depend fmt lint vet test
```

Comment:
nit: I think a better target name here is `ci`.

Reply:
or "verify" or "test"

Reply:
I went with `verify` as that's what k8s calls it.